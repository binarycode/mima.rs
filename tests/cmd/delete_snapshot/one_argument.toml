args = 'delete-snapshot one'
status.code = 2
