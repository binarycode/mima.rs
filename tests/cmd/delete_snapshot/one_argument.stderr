error: The following required arguments were not provided:
    <SNAPSHOT_ID>

USAGE:
    mima delete-snapshot <GUEST_ID> <SNAPSHOT_ID>

For more information try --help
